common/utility.go

```diff
@@ -0,0 +1,14 @@
+package common
+
+import (
+	"encoding/json"
+)
+
+func ValidateJson(s string) (bool, error) {
```

Comment:
So I expected a new test file `utility_test.go` with test for the new `ValidateJson` function